e2e/ansible/playbooks/utils/namespace_task.yml

```diff
@@ -41,4 +37,14 @@
      delay: 30
      retries: 15
 
+   - name: Test Cleanup Passed
+     set_fact:
+        cflag: "Cleanup Passed"
```

Comment:
@vibhor995 , it is possible we might have failed cleanup earlier !! Setting cleanup failed  flag on namespace deletion failure is fine.

Reply:
Yes, i made that changes .